plutus-playground-client/src/Data/Functor/Foldable.purs

```diff
@@ -0,0 +1,42 @@
+module Data.Functor.Foldable where
```

Comment:
this is either 2 modules or should be named something to do with json/playground. Ah, presumably the instances can't go anywhere else since they are for `Fix`, so just a module name change.

Reply:
Ah, I chose that module name to match the Haskell one. Seemed the least-confusing thing to call it: http://hackage.haskell.org/package/recursion-schemes-5.1.3/docs/Data-Functor-Foldable.html#t:Fix

Reply:
ok but I feel like in both cases `Data.*` shouldn't contain any json or schema stuff.

Reply:
I agree, but PS's orphan instances restriction ties my hands a bit here. :-(

Reply:
_grudgingly accepts_

Reply:
we've come across too many places where orphan instances would be beneficial

Reply:
Tragic and true. :-(